modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js

```diff
@@ -0,0 +1,304 @@
+												<div className="autofit-col inline-item-before small">
+													{language.languageId}
+												</div>
+
+												<div className="autofit-col">
+													{language.default && (
+														<ClayLabel
+															className="d-inline"
+															displayType="info"
+														>
+															{Liferay.Language.get(
+																'default'
+															)}
+														</ClayLabel>
+													)}
+												</div>
+											</div>
+										</div>
+
+										<div className="autofit-col">
+											<ClayLink
+												borderless
+												displayType="secondary"
+												href={language.viewURL}
+												monospaced
+												outline
+											>
+												<ClayIcon symbol="view" />
+											</ClayLink>
+										</div>
+									</div>
+								))}
+							</div>
+						)}
+
+						{!!tags.length && (
+							<div className="mb-4 sidebar-dl sidebar-section">
+								<p className="h5">
+									{Liferay.Language.get('tags')}
+								</p>
+
+								<p>
+									{tags.map((tag) => (
+										<ClayLabel
+											displayType="secondary"
+											key={tag}
+										>
+											{tag}
+										</ClayLabel>
+									))}
+								</p>
+							</div>
+						)}
+
+						{!!categories.length && (
+							<div className="mb-4 sidebar-dl sidebar-section">
+								<p className="h5">
+									{Liferay.Language.get('categories')}
+								</p>
+
+								<p>
+									{categories.map((category) => (
+										<ClayLabel
+											displayType="secondary"
+											key={category}
+										>
+											{category}
+										</ClayLabel>
+									))}
+								</p>
+							</div>
+						)}
+
+						{formattedDisplayDate && (
+							<div className="mb-4 sidebar-dl sidebar-section">
+								<p className="h5">
+									{Liferay.Language.get('display-date')}
+								</p>
+
+								<p>{formattedDisplayDate}</p>
+							</div>
+						)}
+
+						{formattedCreateDate && (
+							<div className="mb-4 sidebar-dl sidebar-section">
+								<p className="h5">
+									{Liferay.Language.get('creation-date')}
+								</p>
+
+								<p>{formattedCreateDate}</p>
+							</div>
+						)}
+
+						{formattedModifiedDate && (
+							<div className="mb-4 sidebar-dl sidebar-section">
+								<p className="h5">
+									{Liferay.Language.get('modified-date')}
+								</p>
+
+								<p>{formattedModifiedDate}</p>
+							</div>
+						)}
+
+						{formattedExpirationDate && (
+							<div className="mb-4 sidebar-dl sidebar-section">
+								<p className="h5">
+									{Liferay.Language.get('expiration-date')}
+								</p>
+
+								<p>{formattedExpirationDate}</p>
+							</div>
+						)}
+
+						{formattedReviewDate && (
+							<div className="mb-4 sidebar-dl sidebar-section">
+								<p className="h5">
+									{Liferay.Language.get('review-date')}
+								</p>
+
+								<p>{formattedReviewDate}</p>
+							</div>
+						)}
```

Comment:
You could reduce some of the markup redundancy by creating an array and mapping over

```jsx
{[
	{
		date: formattedExpirationDate,
		title: Liferay.Language.get('expiration-date'),
	},
        // ...
].map(
	({date, title}) =>
		date && (
			<div
				className="mb-4 sidebar-dl sidebar-section"
				key={title}
			>
				<p className="h5">{title}</p>

				<p>{date}</p>
			</div>
		)
)}
```

Not necessary, but might help

Reply:
Pretty cool, will do!